From bad3b2f59482e1092783d5241445329c3fd8b060 Mon Sep 17 00:00:00 2001
From: Pankaj Bansal <pankaj.bansal@nxp.com>
Date: Mon, 22 Apr 2019 16:22:49 +0530
Subject: [PATCH 530/741] arm64: dts: fsl: layerscape: fix warnings when
 compiling dts files

commit 4fef5325d26e3d34fefc2811fd66cd940c6e6a55 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

when compiling dts file using DTC_FLAG='-@', the device tree compiler
reports these warnings:

Warning (simple_bus_reg): /soc/mdio@0x8c0b000: simple-bus unit address
format error, expected "8c0b000"
Warning (unit_address_format): /pfe@04000000: unit name should not have
leading 0s

Fixed the node names to silence these warnings.

Signed-off-by: Pankaj Bansal <pankaj.bansal@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi | 2 +-
 arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
index e0bf50bd3b0a..f42dee1e6d13 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
@@ -515,7 +515,7 @@
                 };
        };
 
-       pfe: pfe@04000000 {
+       pfe: pfe@4000000 {
                 compatible = "fsl,pfe";
                 reg =   <0x0 0x04000000 0x0 0xc00000>,  /* AXI 16M */
                         <0x0 0x83400000 0x0 0xc00000>;  /* PFE DDR 12M */
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
index efbcb44a8c1e..fc97e1bfce6e 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1088a.dtsi
@@ -286,7 +286,7 @@
 		};
 
 		/* TODO: WRIOP (CCSR?) */
-		emdio1: mdio@0x8B96000 { /* WRIOP0: 0x8B8_0000,
+		emdio1: mdio@8B96000 { /* WRIOP0: 0x8B8_0000,
 					  * E-MDIO1: 0x1_6000
 					  */
 			compatible = "fsl,fman-memac-mdio";
@@ -299,7 +299,7 @@
 			#size-cells = <0>;
 		};
 
-		emdio2: mdio@0x8B97000 { /* WRIOP0: 0x8B8_0000,
+		emdio2: mdio@8B97000 { /* WRIOP0: 0x8B8_0000,
 					  * E-MDIO2: 0x1_7000
 					  */
 			compatible = "fsl,fman-memac-mdio";
diff --git a/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
index e64d1c9664b5..e19732ca44fe 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls208xa.dtsi
@@ -512,7 +512,7 @@
 		};
 
 		/* TODO: WRIOP (CCSR?) */
-		emdio1: mdio@0x8B96000 { /* WRIOP0: 0x8B8_0000,
+		emdio1: mdio@8B96000 { /* WRIOP0: 0x8B8_0000,
 					  * E-MDIO1: 0x1_6000
 					  */
 			compatible = "fsl,fman-memac-mdio";
@@ -525,7 +525,7 @@
 			#size-cells = <0>;
 		};
 
-		emdio2: mdio@0x8B97000 { /* WRIOP0: 0x8B8_0000,
+		emdio2: mdio@8B97000 { /* WRIOP0: 0x8B8_0000,
 					  * E-MDIO2: 0x1_7000
 					  */
 			compatible = "fsl,fman-memac-mdio";
-- 
2.17.1

